chainerrl/agents/dqn.py

```diff
@@ -201,28 +201,27 @@ def update(self, experiences, errors_out=None):
 
         This function is thread-safe.
         Args:
-          experiences (list): list of dict that contains
+          experiences (list): list of dicts that contains
```

Comment:
It is `list of lists of dicts` now?

Reply:
Yes, that's right. I have changes to make to the docstrings.